eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryTest.java

```diff
@@ -234,7 +235,7 @@ public void testRegionIsFoundWithMafGreaterThan() {
         regions.add(region);
         List<VariantEntityRepositoryFilter> filters = new FilterBuilder().withMaf(">0.125").build();
         List<String> exclude = new ArrayList<>();
-        testFiltersHelperRegion(regions, filters, exclude, 37);
+        testFiltersHelperRegion(regions, filters, exclude, 43);
```

Comment:
Have these new numbers being checked against direct queries to the database, in order to make sure the tests themselves are correct?